index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>togoville jazz festival homepage</title>
+</head>
+<body>
+     
+    <header>
+        <ul class="header-links">
+            <li><a href="#" class="social-link fa fa-twitter"></a></li>
+            <li><a href="#" class="social-link fa fa-facebook"></a></li>
+            <li><a class="lato-font-black" href="#">English</a></li>
+            <li><a class="lato-font-black" href="#">My page</a></li>
+            <li><a class="lato-font-black" href="#">Logout</a></li>
+        </ul>
+    </header>
+    
+    <main> 
+        <nav class="nav-bar">
+            <div class="hamburger-class">
+                <a href="#" ><img src="./img/iconmonstr-menu-1-240.png" alt="menu button"></a>
+            </div>
+            <ul class="menu-bar">
+                <li class="menu-logo" ><a href="./index.html"><img src="./img/level production icon 3.png" alt="level production icon"></a></li>
+                <li><a class="lato-font-black aboutme-link" href="./aboutme.html">About</a></li>
+                <li><a class="lato-font-black" href="#program">Program</a></li>
+                <li><a class="lato-font-black" href="#">Join</a></li>
+                <li><a class="lato-font-black" href="#sponsor">Sponsor</a></li>
+                <li><a class="lato-font-black" href="#">News</a></li>
+                <li class="campaign"><a class="lato-font-red" href="#">LP campaign</a></li>
+            </ul>
+        </nav> 
```

Comment:
- [ ] Amazing job with the design and the interactions. There are some minor details remaining to match the template. 
I think it is semantically more correct to put the `nav` in the `header` section. Kindly move it to the header section. 🤝️
Please decrease the horizontal padding on the `LP campaign` button to match the design. 😎️
Kindly implement these changes on the About page too.

| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/217796753-4c3513e4-6ff9-4f75-ad32-39bdf7b6983f.png" width="200">|<img src="https://user-images.githubusercontent.com/17243325/217796759-316870df-ab13-4635-b3ff-0f01f22a326d.png" width="200">|